Lib/test/test_random.py

```diff
@@ -430,6 +430,25 @@ def test_bug_27706(self):
             ['0x1.b0580f98a7dbep-1', '0x1.84129978f9c1ap-1',
              '0x1.aeaa51052e978p-2', '0x1.092178fb945a6p-2'])
 
+    def test_bug_31482(self):
+        # Verify that version 1 seeds are unaffected by hash randomization
+        # when the seeds are expressed as bytes rather than strings
+
+        self.gen.seed(b'nofar', version=1)   # hash('nofar') == 5990528763808513177
```

Comment:
> hash('nofar') == 5990528763808513177

This is not true because of hash randomization.

Reply:
The comment referred to the Python2.7 hash() function (non-randomized).  That is what is being emulated by version 1.

Reply:
Could you add a comment for this?